### ISSUE TITLE
Unable to build .NET Core/Standard projects in VS 2019

### ISSUE THREAD

**Issue by olivegamestudio** [dotnet/core#3615]
When building the solution, the .NET core/standard projects do *not* build and the output shown is the following. It effectively is skipping projects.

------ Skipped Rebuild All: Project: XXX ------
========== Rebuild All: 0 succeeded, 0 failed, 1 skipped ==========

Repairing VS did not fix this.

**Comment by warappa** [comment]
@olivegamestudio A guess: maybe in the solution's build configuration the "Build"-checkbox for project XXX is not checked?

**Comment by olivegamestudio** [comment]
> @olivegamestudio A guess: maybe in the solution's build configuration the "Build"-checkbox for project XXX is not checked?

I wish that was the case, but alas all projects were checked for building.

**Comment by scalablecory** [comment]
This sounds like something VS support will need to resolve, but @safern @ViktorHofer might have some pointers to help diagnose the issue.

**Comment by ViktorHofer** [comment]
Can you provide a minimal repo?

**Comment by vova1987** [comment]
Same here. after converting most of my projects to netcore3 build target - all projects are skipped during the build

**Comment by nefcanto** [comment]
Such a frustrating experience. I upgraded to 3.1 and now I have the same issue here. I tried to delete everything and get specific version from source control (Azure DevOps). No result. I checked and all projects are marked for build with Any CPU configuration. I increased the build verbosity to Diagnostics, and for skipped projects it made no difference. 

How can we diagnose it?

@scalablecory how can we create a minimal repo? This is happening only to some of our projects and we don't know the reason. I create the exactly same project somewhere else and it gets built. What repo can I make?
